benchmarks/src/main/scala/zio/IODeepFlatMapBenchmark.scala

```diff
@@ -122,10 +122,10 @@ class IODeepFlatMapBenchmark {
 
   private[this] def zioDeepFlatMap(runtime: Runtime[Any]): BigInt = {
     def fib(n: Int): UIO[BigInt] =
-      if (n <= 1) ZIO.succeedLazy[BigInt](n)
+      if (n <= 1) ZIO.succeed[BigInt](n)
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
Fixed